doc/nrf/libraries/zigbee/lib_zigbee_shell.rst

```diff
@@ -0,0 +1,662 @@
+﻿.. _lib_zigbee_shell:
+
+Zigbee shell
+############
+
+The Zigbee shell library implements a set of Zigbee shell commands that allow you to control the device behavior and perform operations in the Zigbee network, such as steering the network, discovering devices, and reading the network properties.
+Enabling a device to work with these commands simplifies testing and debugging of an existing network and reduces the amount of reprogramming operations.
+
+.. _zigbee_shell_extending_samples:
+
+Adding support for Zigbee shell commands
+****************************************
+
+The Zigbee shell commands are implemented using Zephyr's :ref:`zephyr:shell_api` interface.
+By default, it uses the UART backend.
+To change this and other Zephyr's shell settings (for example, the prompt or the maximum amount of accepted command arguments), check the documentation page in Zephyr.
+
+Zigbee shell command support can be added to any Zigbee sample.
+Some of the commands use an endpoint to send packets from, so no endpoint handler is allowed to be registered for this endpoint.
+
+To extend a sample with the Zigbee shell command support, set the following KConfig options:
+
+* ``CONFIG_ZIGBEE_SHELL`` - This option enables Zigbee shell and Zephyr's :ref:`zephyr:shell_api`.
+* ``CONFIG_ZIGBEE_SHELL_ENDPOINT`` - This option specifies the endpoint number to be used by the Zigbee shell instance.
+  Endpoint must be present at the device and you must not register an endpoint handler for this endpoint.
+* ``CONFIG_ZIGBEE_SHELL_DEBUG_CMD`` - This option enables commands useful for testing and debugging.
+
+  .. note::
+    Using debug commands can make the device unstable.
+
```

Comment:
@sebastiandraus , shouldn't we add a note here about ``CONFIG_ZIGBEE_SHELL_LOG_LEVEL``? I can imagine people wondering what the relation is between these options.